users/klackygears/klackygears.h

```diff
@@ -0,0 +1,92 @@
+#ifndef USERSPACE
+#define USERSPACE
+
+#include "quantum.h"
+#include "wrappers.h"
+
+#pragma once
+#include "quantum.h"
+#include "version.h"
+#include "eeprom.h"
+
```

Comment:
```suggestion
#pragma once 

#include QMK_KEYBOARD_H

#include "wrappers.h"
#include "eeprom.h"
```

Also, the `version.h` should be included only where it's needed, otherwise, every file that includes it will be recompiled  every time (as this changes when compiling)